src/Compiler/Facilities/CompilerLocation.fs

```diff
@@ -123,6 +123,7 @@ module internal FSharpEnvironment =
             |]
         elif typeof<obj>.Assembly.GetName().Name = "System.Private.CoreLib" then
             [|
+                "net10.0"
```

Comment:
This line is the reason for IlVerify change, since it shifts rest of the file by 1 line of code.

Reply:
Ah, right, I forgot that I've changed it.